hudi-client/src/main/java/org/apache/hudi/table/rollback/RollbackExecutor.java

```diff
@@ -69,7 +69,7 @@ public RollbackExecutor(HoodieTableMetaClient metaClient, HoodieWriteConfig conf
    * Performs all rollback actions that we have collected in parallel.
    */
   public List<HoodieRollbackStat> performRollback(JavaSparkContext jsc, HoodieInstant instantToRollback,
-      List<RollbackRequest> rollbackRequests) {
+                                           List<RollbackRequest> rollbackRequests) {
```

Comment:
Is the original indent the right one?

Reply:
yes.. will revert. dont know how it got changed